atlasdb-conjure/src/main/java/com/palantir/atlasdb/http/v2/DefaultTargetFactory.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.http.v2;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.atlasdb.config.AuxiliaryRemotingParameters;
+import com.palantir.atlasdb.config.ImmutableAuxiliaryRemotingParameters;
+import com.palantir.atlasdb.config.ServerListConfig;
+import com.palantir.atlasdb.http.TargetFactory;
+import com.palantir.conjure.java.client.config.ClientConfiguration;
+import com.palantir.conjure.java.client.jaxrs.JaxRsClient;
+import com.palantir.conjure.java.config.ssl.TrustContext;
+import com.palantir.conjure.java.dialogue.serde.DefaultConjureRuntime;
+import com.palantir.conjure.java.okhttp.HostMetricsRegistry;
+import com.palantir.dialogue.Channel;
+import com.palantir.dialogue.ConjureRuntime;
+import com.palantir.dialogue.hc4.ApacheHttpClientChannels;
+import com.palantir.logsafe.SafeArg;
+import com.palantir.logsafe.exceptions.SafeIllegalArgumentException;
+import com.palantir.logsafe.exceptions.SafeIllegalStateException;
+import com.palantir.util.CachedTransformingSupplier;
+
+public enum DefaultTargetFactory implements TargetFactory {
+    INSTANCE;
+
+    private static final HostMetricsRegistry HOST_METRICS_REGISTRY = new HostMetricsRegistry();
```

Comment:
might as well just use a `NoOpHostEventsSink` if they're never wiring this up to anything...

Reply:
ya, I started working on wiring this up at some point last month